src/components/matrixSize.jsx

```diff
@@ -17,13 +17,14 @@ export default class MatrixSize extends React.Component {
       this.setState({ numCols: event.target.value });
   }
   handleClick(){
-  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(this.state.numRows);
-  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(this.state.numCols);
+  firebase.database().ref('grids/' + this.state.gridId + '/numRows' ).set(parseInt(this.state.numRows));
+  firebase.database().ref('grids/' + this.state.gridId + '/numCols' ).set(parseInt(this.state.numCols));
 
    this.props.updateGrid(this.state.gridId);
   }
   componentDidMount(){
     this.setState({gridId: this.props.gridId});
+    console.log(this)
```

Comment:
This will probably be handled in the other review, but please remove the console log.